packages/typespec-ts/static/static-helpers/urlTemplate.ts

```diff
@@ -0,0 +1,195 @@
+//---------------------
```

Comment:
Can this file be formatted with the standard 2 spaces indentation

Reply:
@v-jiaodi could you have a seprated pr to include format for test and static files also?

```
"prettier": "prettier --config ./.prettierrc \"(src|static|test)/**/*.!(d).ts\"",
```

Reply:
https://github.com/Azure/autorest.typescript/pull/3094